Neo4j.Driver/Neo4j.Driver.Tests.Integration/Internals/IntegrationTestAttribute.cs

```diff
@@ -295,6 +299,32 @@ public sealed class RequireServerWithIPv6FactAttribute : RequireServerFactAttrib
             {
                 Skip = "IPv6 is disabled";
             }
+
+            if (string.IsNullOrEmpty(Skip))
```

Comment:
This seems a little dangerous to me.
If we would fail the test because we mistakenly made it so we can't resolve an address it would be skipped and the suite would pass.